.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Executables
+on: push
+jobs:
+  lint:
+    name: Build Executables
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1       
+      - uses: MOZGIII/install-ldid-action@v1
+        with:
+          tag: v2.1.5-procursus2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - run: ./scripts/build.sh
+      - uses: actions/upload-artifact@v3
+        with:
+          name: macos executable
+          path: run-macos
+      - uses: actions/upload-artifact@v3
+        with:
+          name: linux executable
+          path: run-linux
+      - uses: actions/upload-artifact@v3
+        with:
+          name: win executable
+          path: run-win.exe
```

Comment:
we need to package both `arm64, amd64` variant for unix 

Eg: https://github.com/github/gh-ost/releases

Also the name of the package should be `percy`

Reply:
while I was testing the executables on my machine. 

`./run-macos` works without any issues, though

if I run using absolute path, it breaks.

Reply:
One more thing, lets rename executables correctly. it should be 

`percy` for unix/linux and `percy.exe` on windows

Reply:
as decided we will update docs. if another cli is installed at that place it might be reason for it to break